src/pages/Detail/index.tsx

```diff
@@ -135,7 +138,13 @@ export const DetailPage = () => {
             <IconButton
               aria-label="+"
               icon={<AddIcon />}
-              onClick={() => setProductCount(productCount + 1)}
+              onClick={() => {
+                if (productCount < giftOrderLimit) {
+                  setProductCount(productCount + 1);
```

Comment:
```suggestion
                  setProductCount(v => v + 1);
```
이 부분도 이럲게 한 번 표현해보세요!